projects/laji/src/app/shared-modules/spreadsheet/service/mapping.service.ts

```diff
@@ -338,7 +338,7 @@ export class MappingService {
       return value.toISOString();
     }
 
-    return String(value);
+    return String(value || '');
```

Comment:
This would replace `"0"` with `""`. How about `value ?? ''` ?

Reply:
I didn't read the whole code context. In this problem domain, the value shouldn't be `"0"` ever anyway.

Reply:
nw :)